### DISCUSSION TITLE
查找資料工具分享

### DISCUSSION THREAD

**Discussion by xkllkx** [1101-GS4719-C/Python-Community-GS4719-C#427] [General]
每次電腦資料太多、忘記資料放哪裡的時候，總會利用windows內建的檔案搜尋功能，但它真的太慢了，尤其是大範圍搜尋時，如果換作使用Everything工具來搜尋的話，真的能減少很多時間，激推。
[Everything](https://www.voidtools.com/)

**Comment by pekoranoinu** [comment]
請問樓主這個是在業配嗎(O-O)?

**Reply by xkllkx** [reply]
沒有沒有，純分享而已

**Reply by chupochun** [reply]
內建的搜尋真的是有夠慢，找個檔案都可以泡一碗泡麵了

**Reply by chia-kai-chang** [reply]
我都用這個Everything軟體，很好用，安裝時記得把「開機啟動Eveerything」關閉

**Comment by minniewei** [comment]
這個超級好用的！！我現在也都用它

**Comment by henrywen547** [comment]
雖然之前有聽過這個，但沒去裝
剛剛裝完發現真是太方便了

**Comment by Rosancl** [comment]
推 內建搜尋真的慢

**Comment by d26488399** [comment]
感謝分享 內建的搜尋真的不知道是在慢幾點的

**Comment by awkward-willy** [comment]
感謝樓主的分享，在實際安裝並使用後發覺速度真的快很多
完美的解決我常常忘記檔案放在甚麼地方的困擾

**Comment by Lai27330** [comment]
太方便了的功能了，終於可以解決我常常下載完檔案就找不到位置的困擾了，感謝分享。

**Comment by HsuChen88** [comment]
第一次聽到 謝謝分享 馬上下載

**Comment by sys511613** [comment]
剛剛試用了發現找東西的速度真的很快
不會像windows內建的在那邊轉圈圈半天找不到
感謝分享

**Comment by 11timyyy** [comment]
終於可以快速地找到想找的資料了!

**Comment by bloodtrail0511** [comment]
內建的搜尋常常慢到我都已經找到了結果電腦還沒找到XD

**Comment by yucheng1028** [comment]
內建搜尋真的是trash

**Comment by SamTU0222** [comment]
內建搜尋真的很慢，尤其當筆電又不是SSD時又慢上一截

**Comment by yenwei12138** [comment]
確實每次找檔案都搜尋很久，會想載來試試看
